README.md

```diff
@@ -1,108 +1,28 @@
 # Todo App Java On Azure
 
-This TodoList app is an Azure Java application. It provides end-to-end CRUD operation to todo list item from front-end AngularJS code. Behind the scene, todo list item data store is [Azure CosmosDB DocumentDB](https://docs.microsoft.com/en-us/azure/cosmos-db/documentdb-introduction). This application uses [Azure CosmosDB DocumentDB Spring Boot Starter](https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter) and AngularJS to interact with Azure. This sample application provides several deployment options to deploy to Azure, pls see deployment section below. With Azure support in Spring Starters, maven plugins and Eclipse/IntelliJ plugins, Azure Java application development and deployment is effortless now.
-
-
-## TOC
-
-* [Requirements](#requirements)
-* [Create Azure Cosmos DB documentDB](#create-azure-cosmos-db-documentdb)
-* [Configuration](#configuration)
-* [Run it](#run-it)
-* [Contribution](#contribution)
-* Add new features
-    * [Add AAD](https://github.com/Microsoft/todo-app-java-on-azure/tree/aad-start)
-    * [Add KeyVault]()
-* Deployment
-    * [Deploy to Azure Web App for Containers using IntelliJ plugin](./doc/deployment/deploy-to-azure-web-app-using-intelliJ-plugin.md)
-    * [Deploy to Azure Web App for Containers using Eclipse plugin](./doc/deployment/deploy-to-azure-web-app-using-eclipse-plugin.md)
-    * [Deploy to Azure Container Service Kubernetes cluster using Maven plugin](./doc/deployment/deploy-to-azure-container-service-using-maven-plugin.md)
-    * [Deploy to Azure Web App for Containers using Maven plugin](./doc/deployment/deploy-to-azure-web-app-using-maven-plugin.md)
-* [Useful link](#useful-link)
+This TodoList app is an Azure Java SpringBoot application. It provides end-to-end CRUD operations on todo list items from front-end AngularJS code. Behind the scenes, the todo list item data store is [Azure CosmosDB DocumentDB](https://docs.microsoft.com/en-us/azure/cosmos-db/documentdb-introduction). This application uses [Azure CosmosDB DocumentDB Spring Boot Starter](https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter) and AngularJS to interact with Azure. In particular, this version of the application shows how to use [Azure Application Insights](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-overview) to monitor the performance and usage of a Java SpringBoot application. For more information on how to use Application Insights in other types of Java applications, refer to [Get started with Application Insights in a Java web project] (https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started).
 
 ## Requirements
-
-* [JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 1.8 and above
```

Comment:
those steps are still necessary for user to finish a complete end to end todo app with application insight. please add steps of application insight into original steps, instead of overwrite it. thanks.